[docs] Added examples for COPY in YSQL (#5448)
yugabyte/yugabyte-db: docs/content/latest/api/ysql/commands/cmd_copy.md

Use the `COPY` statement to transfer data between tables and files. `COPY TO` copies from tables to files. `COPY FROM` copies from files to tables. `COPY` outputs the number of rows that were copied.

> {{< note title="Note" >}}
> 
> `COPY` works with files that reside local to the yb-tserver that we're connecting to. To work with files that reside
>  on the client, use `\copy` in [`ysqlsh` cli](../../../../admin/ysqlsh#copy-table-column-list-query-from-to-filename-program-command-stdin-stdout-pstdin-pstdout-with-option).
> {{< /note >}}

Reviewer: ```suggestion
{{< note title="Note" >}}

The `COPY` statement can be used with files residing locally to the YB-TServer that you connect to. To work with files that reside on the client, use `\copy` in [`ysqlsh` cli](../../../../admin/ysqlsh#copy-table-column-list-query-from-to-filename-program-command-stdin-stdout-pstdin-pstdout-with-option).

{{< /note >}}
```

## Examples

> Assuming we have a table like below:

Reviewer: ```suggestion
The examples below assume a table like this:
```

```postgresql
yugabyte=# CREATE TABLE users(id BIGSERIAL PRIMARY KEY, name TEXT);
yugabyte=# INSERT INTO users(name) VALUES ('John Doe'), ('Jane Doe'), ('Dorian Gray');
yugabyte=# SELECT * FROM users;
 id |    name     
----+-------------
  3 | Dorian Gray
  2 | Jane Doe
  1 | John Doe
(3 rows)
```

> ### Export the whole table
> Copy the whole table to a comma separated CSV file with column names in the header.

Reviewer: ```suggestion
### Export an entire table

Copy the entire table to a CSV file, with column names in the header.

```

Reviewer: Say where the file is exported to.

Author: mentioned "absolute path"

```postgresql
yugabyte=# COPY users TO '/home/yuga/Desktop/users.txt.sql' DELIMITER ',' CSV HEADER;
```

> ### Export partial table using WHERE clause with column selection
> Use a WHERE clause to filter rows and only `name` column.

Reviewer: ```suggestion
### Export a partial table using the WHERE clause with column selection

In the following example, a `WHERE` clause is used to filter the rows and only the `name` column.

```

```postgresql
yugabyte=# COPY (SELECT name FROM users where name='Dorian Gray') TO '/home/yuga/Desktop/users.txt.sql' DELIMITER
 ',' CSV HEADER;
```

> ### Importing the previous exports

Reviewer: ```suggestion
### Import from CSV files

In the following example, the data exported in the previous examples are imported.

```

Reviewer: Mention where the data is imported. If you're copying the data to a different table, mention it.

Author: mentioned it

```postgresql
yugabyte=# COPY users FROM '/home/yuga/Desktop/users.txt.sql' DELIMITER ',' CSV HEADER;
```

> - Errors are raised if the table does not exist.
> - `COPY TO` can only be used with regular tables.
> - `COPY FROM` can be used with tables, foreign tables, or views.

Reviewer: ```suggestion
- If the table does not exist, errors are raised.
- `COPY TO` can only be used with regular tables.
- `COPY FROM` can be used with tables, foreign tables, and views.
```
